rust-fsm-dsl/Cargo.toml

```diff
@@ -8,7 +8,7 @@ readme = "../README.md"
 license = "MIT"
 categories = ["data-structures", "rust-patterns"]
 keywords = ["fsm"]
-version = "0.6.1"
+version = "0.6.0"
```

Comment:
I removed my version number increment. Thats probably something the maintainer should do.